gradle/libs.versions.toml

```diff
@@ -38,6 +38,8 @@ jcodec-javase = { module = "org.jcodec:jcodec-javase", version.ref = "jcodec" }
 kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
 kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.7.1" }
 
+ktlint = { module = "com.pinterest.ktlint:ktlint-rule-engine", version.ref = "ktlint" }
```

Comment:
This is to help Renovate understand the "ktlint" version so it can upgrade it from mavenCentral.

Another, potentially better, long term solution is to remove the `libs.versions.ktlint` constant fully and rely on the default built-in version in Spotless. It's pretty hard to get incompatible if there's no override. _This might become a non-issue, because it looks like ktlint is preparing for a 1.0 release (as far as I understand they pre-broke users, so 1.0 doesn't break users 🙄)._